src/app/components/internal/SideNav/SideNav.js

```diff
@@ -259,119 +259,145 @@ class SideNav extends Component {
-              target="_blank"
-              role="button"
-              iconDescription="sidenav link icon"
-              onClick={e => this.handleClick(e, 'Developer Kit')}
-            >
-              Vanilla Repo
-            </Button>
-            <Button
-              href="https://github.com/carbon-design-system/carbon-components-react"
-              className="side-nav__link bx--btn"
-              kind="secondary"
-              icon="arrow--right"
-              target="_blank"
-              role="button"
-              iconDescription="sidenav link icon"
-              onClick={e => this.handleClick(e, 'React Developer Kit')}
+          <GlobalSearch
+            results={this.state.results}
+            relatedResults={this.state.relatedResults}
+            val={this.state.val}
+            currentQuery={this.state.currentQuery}
+            activeSearch={this.state.activeSearch}
+          />
+          <div className={bottomClasses}>
+            <ul
+              role="menu"
+              aria-label="Page main menu"
+              className="side-nav__main-nav"
+              onClick={e => this.handleClick(e, 'Left Nav')}
             >
-              React Repo
-            </Button>
+              {navItems}
+            </ul>
+            <div className="side-nav__links">
+              <Button
+                href="https://github.com/carbon-design-system/carbon-design-kit"
+                className="side-nav__link bx--btn"
+                kind="secondary"
+                icon="arrow--right"
+                target="_blank"
+                role="button"
+                iconDescription="sidenav link icon"
+                onClick={e => this.handleClick(e, 'Design Kit')}
+              >
+                Design Kit
+              </Button>
+              <Button
+                href="https://github.com/carbon-design-system/carbon-components"
+                className="side-nav__link bx--btn"
+                kind="secondary"
+                icon="arrow--right"
+                target="_blank"
+                role="button"
+                iconDescription="sidenav link icon"
+                onClick={e => this.handleClick(e, 'Developer Kit')}
+              >
+                Vanilla Repo
+              </Button>
+              <Button
+                href="https://github.com/carbon-design-system/carbon-components-react"
+                className="side-nav__link bx--btn"
+                kind="secondary"
+                icon="arrow--right"
+                target="_blank"
+                role="button"
+                iconDescription="sidenav link icon"
+                onClick={e => this.handleClick(e, 'React Developer Kit')}
+              >
+                React Repo
+              </Button>
+            </div>
           </div>
-        </div>
-      </nav>
-    );
+        </nav>
+      );
+    }
   }
-}
 
-export default SideNav;
+  export default SideNav;
+
+
+            // <Button
```

Comment:
Did you intend to remove these lines...?

Reply:
No I didn't. Let me see if I can add it back in.